src/testUtils/mocks/entities.ts

```diff
@@ -2545,3 +2550,16 @@ export function getCorporateActionExistsStub(exists?: boolean): SinonStub {
   }
   return corporateActionExistsStub;
 }
+
+/**
+ * @hidden
+ * Retrieve the stub of the `DividendDistribution.getParticipant` method
+ */
+export function getDividendDistributionGetParticipantStub(
+  getParticipant?: DistributionParticipant
```

Comment:
Should accept a Partial object and merge with the defaults before passing it to `resolves`